.nvmrc

```diff
@@ -0,0 +1 @@
+4.2.6
```

Comment:
I see. Thanks for clarification.  I guess I can see cases where a developer working on this project may want different settings. I just pushed a commit with `.gitignore` and `.nvmrc`.